packages/flutter/test/material/search_test.dart

```diff
@@ -771,6 +771,17 @@ void main() {
       expect(textField.style!.color, themeData.textTheme.bodyText1!.color);
       expect(textField.style!.color, isNot(equals(themeData.primaryColor)));
   });
+
+  // Regression test for: https://github.com/flutter/flutter/issues/78144
+  testWidgets('Leading and Actions widgets can be null', (WidgetTester tester) async {
+    final _TestEmptySearchDelegate delegate = _TestEmptySearchDelegate();
+    await tester.pumpWidget(TestHomePage(delegate: delegate));
+
+    await tester.tap(find.byTooltip('Search'));
+    await tester.pumpAndSettle();
+
+    expect(tester.takeException(), isNull);
```

Comment:
Does this just mean that no exceptions were thrown? Maybe add one more step to the test here to show that the search delegate page is displayed instead of checking that no exceptions were thrown. Then, add a comment saying something like "The search delegate page is displayed with no issues even with a null return values for [buildLeading] and [buildActions]".

Reply:
make sense